libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/OfflineNavigator.kt

```diff
@@ -20,12 +26,23 @@ internal class OfflineNavigator @JvmOverloads constructor(
      * can be called safely
      */
     fun configure(tilePath: String, callback: OnOfflineTilesConfiguredCallback) {
-        ConfigureRouterTask(
-            navigator,
+        val endPointConfig = TileEndpointConfiguration(
+            host,
+            version,
+            accessToken,
+            userAgent,
+            navigatorVersion
```

Comment:
Why does NN need `navigatorVersion`? I'm asking because we're currently passing empty `""`  🤔 

cc @vmihaylenko

Reply:
no idea

Reply:
added comment below